### DISCUSSION TITLE
flet是否可以加入Rive的动画设计组件扩展

### DISCUSSION THREAD

**Discussion by AERCR3** [flet-dev/flet#1959] [Ideas]
您看，看到python有flet的项目，真的很高兴，但是似乎flet还不够成熟，似乎开发者打算通过增加组件来让Flet变的强大，但是似乎可以换一种方式，我看到Rive基于状态机的控件控制方式非常先进的，用户可以使用它们的编辑工具，免费构建拥有状态机切换的UI控件，并且可以集成的Flutter以及其他众多的游戏引擎中，而且看似Flutter似乎现在也有rive中内部集成，这样可以更好的构建用户界面免去flet的开发者在flet中加入组件的大量工作，集成Rive就可以拥有无限的动态机组件和丰富的美术资源。希望这个建议可以大量减少Flet开发者大量不必要的同质化的工作，也希望可以在Flet中看到更多的创新和思考。谢谢，感谢Flet的开发者，为我们做的努力和贡献，大家有什么新的想法可以留言联系

**Comment by ndonkoHenri** [comment]
#89
